### ISSUE TITLE
Great stuff

### ISSUE THREAD

**Issue by nonword** [lizstarin/autocover#1]
Just in case you continue working on this and are curious about querying containers directly:

So, we only recently began surfacing collections and containers in the Digital Collections site. The data for these groupings exists in the repo powering the API, but the API is quiet about it. If you find a container you like in DC, there's a corresponding API endpoint for retrieving those images. For example, this lovely container:

http://digitalcollections.nypl.org/collections/ornament-and-pattern-pre-victorian-to-art-deco#/?tab=navigation&roots=6:a24a7190-c62c-012f-a804-58d385a7bc34&page=1

.. Has the following API endpoint:

http://api.repo.nypl.org/api/v1/items/a24a7190-c62c-012f-a804-58d385a7bc34

Basically, take the UUID in the roots param (removing the suffix before the colon - "6:" in the example above) and put it on the API URL as above.

Unfortunately that technique only works for containers, but hopefully that helps narrow in on certain sets of content.
